src/Scripts/Game/Feature/Quantity/EL_QuantityComponent.c

```diff
@@ -0,0 +1,261 @@
+class EL_QuantityComponentClass : ScriptComponentClass
+{
+	override static array<typename> CannotCombine(IEntityComponentSource src)
+	{
+		return {EL_QuantityComponent}; // Prevent user from adding multiple quantity components
+	}
+
+	[Attribute(defvalue: "1", desc: "Inclusive maximal quantity this item can stack to. 0 means unlimited stack size.")]
+	int m_iMaxQuantity;
```

Comment:
Not sure if this is a convention, but I would put attribute vars at the top

Reply:
Indeed!